### PR TITLE
Update tutor logo

### PR DESCRIPTION
**Pull request by g-re-g**
I noticed while running the tutor that the logo in the tutor was a little out of sync on with the logo on the website and in the repository. I adjusted the letter proportions and added the helix icon to the logo. Lemme know what y'all think. Happy to change it however folks like or delete PR if no changes wanted!

Screen shot of the proportion fitting:
Screenshot 2023-01-25 at 3 46 23 PM

Screenshot in running helix:
Screenshot 2023-01-27 at 11 13 11 AM

### CODE REVIEW

**Review by the-mikedavis — COMMENTED** [review]
this is cool!

**Comment by archseer** [comment]
This is very cool 🎉 ! Did you do this manually?

File: runtime/tutor [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+          ,v|||||+'.      _     _           _
+       ,v|||||^'>####    | |   | |   ___   | | (_) __  __
+      |||||^'  .v####    | |___| |  /   \  | |  _  \ \/ /
+      ||||=..v#####P'    |  ___  | /  ^  | | | | |  \  /
+      ''v'>#####P'       | |   | | |  ---  | | | |  /  \
+      ,######/P||x.      |_|   |_|  \___/  |_| |_| /_/\_\
+      ####P' "x|||||,
+      |/'       'x|||    A post-modern modal text editor.
+       '           '|
+
```
[the remark this answers is not in the document]
Comment (the-mikedavis) [reply runtime/tutor:14]: That looks great!

File: runtime/tutor [hunk 2]
```diff
@@ -20,6 +36,13 @@ _________________________________________________________________
  the first lesson.
 
 
+
```
Comment (the-mikedavis) [inline runtime/tutor:39]:
```suggestion
```

looks like this section is just 1 column too tall

**Review by the-mikedavis — APPROVED** [review]
This looks great, thanks!

**Comment by yashpalgoyal1304** [comment]
Awesome work for the logo - it's looking really pretty, I was dreaming about this ever since hx got a new logo
* widening of H is also good
* but i am not liking the rewriting of other letters of the name.

- the larger X (older) makes sense as the invocation command is `hx`
- the small height of `eli` was looking consistent and cute
- it was forming a wave like pattern:
`large small medium small large`

the new design just feels lacking any character or charm of its own, and totally confuses the eye due to lack of any pattern.

**Comment by archseer** [comment]
I think it looks great and matches the design. The previous version was just quickly generated using an online tool.

If you think you can do better, open a PR.

**Pull request merged**